src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -6965,20 +7141,33 @@ extends System.Object
 
     .method public final hidebysig virtual instance string ToString () cil managed
     {
-        IL_0000: ldnull
-        IL_0001: throw
+        IL_0000: ldstr ""A""
+        IL_0001: ret
     }
 }
 ";
             var source = @"
+var b = new B();
+System.Console.Write(b.ToString());
```

Comment:
Did you mean to execute code in success branch below?

Reply:
Oops, yes I did.

Reply:
Done